### PR TITLE
Feature a help panel to describe what campaigns are for

### PR DESCRIPTION
**Pull request by flamerohr**
Also, it is extensible for later to add links to userhelp docs

screen shot 2017-10-31 at 3 29 24 pm
screen shot 2017-10-31 at 3 29 34 pm

*NOTE*: the links and buttons do not show in the pull request, they're there for preview purposes

Requires https://github.com/silverstripe/silverstripe-admin/pull/333 for new icons added

### CODE REVIEW

**Comment by tractorcow** [comment]
Looks good, but please see my comment on the admin pr.

File: client/src/state/campaign/CampaignReducer.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
+// retrieve showMessage state from user's localStorage
+const showMessage = window.localStorage.getItem('campaign.showMessage');
 const initialState = deepFreeze({
   campaignId: null,
   changeSetItemId: null,
   isPublishing: false,
   view: null,
   newItem: null,
+  showMessage: showMessage === null,
```
Comment (tractorcow) [inline client/src/state/campaign/CampaignReducer.js]: We should only show message if showMessage == null && there are no campaigns; I have campaigns loaded, and my initial load displayed the help message when I didn't expect it.

**Comment by tractorcow** [comment]
On second thoughts, I don't think it's that important. Let's just get this in as is (once we fix the icons).

File: client/src/state/campaign/CampaignReducer.js (hunk 1 above)
Comment (flamerohr) [reply client/src/state/campaign/CampaignReducer.js]: To elaborate:
The trouble with adding the logic to check for campaigns - the `showMessage` state becomes ternary and ambiguous until the list of Campaign records has been received, meaning a few extra layers for something that is stored in `localStorage` after the first time it's closed anyway.

P.S. Great that we agree that it's not that important.

**Comment by newleeland** [comment]
Could we add more padding? (padding: 32px 30px 30px 30px;)
screen shot 2017-11-01 at 2 19 36 pm


Could we remove the right margin on mobile view?
screen shot 2017-11-01 at 1 43 53 pm

**Comment by flamerohr** [comment]
Padding and margins amended

**Pull request merged**